src/backend/opsdispatch.rs

```
@@ -147,6 +147,15 @@ pub(crate) fn do_mkdir(out: &mut impl Write, ops: &mut Ops, parent: &str, name:
     out.flush().ok();
 }
 
+// Permanently delete files (used in Trash directory).
+pub(crate) fn do_permanent_delete(out: &mut impl Write, paths: Vec<String>) {
+    let path_bufs: Vec<PathBuf> = paths.iter().map(|p| PathBuf::from(p)).collect();
```

Clippy (1.97.1)

[warning] redundant closure

(warning)